Citations
---------

.. _citations:
If you use mitoBench, please cite

(not published yet)

The project URL is:

https://github.com/apeltzer/MitoBench

Tools & Methods
~~~~~~~~~~~~~~~

* 1000 Genomes Project Consortium, 2015. A global reference for human genetic variation. Nature, 526(7571), p.68.
* Excoffier, L. and Lischer, H.E., 2010. Arlequin suite ver 3.5: a new series of programs to perform population genetics analyses under Linux and Windows. Molecular ecology resources, 10(3), pp.564-567.
* Hudson, R.R., Slatkin, M. and Maddison, W.P., 1992. Estimation of levels of gene flow from DNA sequence data. Genetics, 132(2), pp.583-589.
* Lischer, H.E. and Excoffier, L., 2011. PGDSpider: an automated data conversion tool for connecting population genetics and genomics programs. Bioinformatics, 28(2), pp.298-299.
* Reynolds, John, Bruce S. Weir, and C. Clark Cockerham. "Estimation of the coancestry coefficient: basis for a short-term genetic distance." Genetics 105.3 (1983): 767-779.
* Slatkin, Montgomery. "A measure of population subdivision based on microsatellite allele frequencies." Genetics 139.1 (1995): 457-462.
* van Oven, M., 2015. PhyloTree Build 17: Growing the human mitochondrial DNA tree. Forensic Science International: Genetics Supplement Series, 5, pp.e392-e394.
* Weissensteiner, H., Pacher, D., Kloss-Brandstätter, A., Forer, L., Specht, G., Bandelt, H.J., Kronenberg, F., Salas, A. and Schönherr, S., 2016. HaploGrep 2: mitochondrial haplogroup classification in the era of high-throughput sequencing. Nucleic acids research, 44(W1), pp.W58-W63.

