% --------------------------------------------------------------------------- %
% Poster for the ECCS 2011 Conference about Elementary Dynamic Networks.      %
% --------------------------------------------------------------------------- %
% Created with Brian Amberg's LaTeX Poster Template. Please refer for the     %
% attached README.md file for the details how to compile with `pdflatex`.     %
% --------------------------------------------------------------------------- %
% $LastChangedDate:: 2011-09-11 10:57:12 +0200 (V, 11 szept. 2011)          $ %
% $LastChangedRevision:: 128                                                $ %
% $LastChangedBy:: rlegendi                                                 $ %
% $Id:: poster.tex 128 2011-09-11 08:57:12Z rlegendi                        $ %
% --------------------------------------------------------------------------- %
\documentclass[a0paper,portrait]{baposter}
\usepackage{relsize}		% For \smaller
\usepackage{url}			% For \url
\usepackage{epstopdf}	% Included EPS files automatically converted to PDF to include with pdflatex
\usepackage{lipsum}	% generate non sense text to fill boxes
\usepackage[export]{adjustbox}
\usepackage{enumitem}
\usepackage[none]{hyphenat}

%%% Global Settings %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\graphicspath{{pix/}}	% Root directory of the pictures
\tracingstats=2			% Enabled LaTeX logging with conditionals

%%% Color Definitions %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\definecolor{bordercol}{RGB}{40,40,40}
\definecolor{headercol1}{RGB}{28,144,153}
\definecolor{headercol2}{RGB}{80,80,80}
\definecolor{headerfontcol}{RGB}{0,0,0}
\definecolor{boxcolor}{RGB}{255,255,255} % white

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% Utility functions %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

%%% Save space in lists. Use this after the opening of the list %%%%%%%%%%%%%%%%
\newcommand{\compresslist}{
	\setlength{\itemsep}{1pt}
	\setlength{\parskip}{0pt}
	\setlength{\parsep}{0pt}
}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%% Document Start %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\begin{document}
\typeout{Poster rendering started}

%%% Setting Background Image %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\background{
%	\begin{tikzpicture}[remember picture,overlay]%
%	\draw (current page.north west)+(-2em,2em) node[anchor=north west]
%	{\includegraphics[height=1.1\textheight]{figures/background}};
%	\end{tikzpicture}
%}

%%% General Poster Settings %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%% Eye Catcher, Title, Authors and University Images %%%%%%%%%%%%%%%%%%%%%%
\begin{poster}{
	grid=false,
	columns=5,
	% Option is left on true though the eyecatcher is not used. The reason is
	% that we have a bit nicer looking title and author formatting in the headercol
	% this way
	%eyecatcher=false,
	borderColor=bordercol,
	headerColorOne=headercol1!70,%blue!20,
	headerColorTwo=headercol1!70,%blue!20,
	headerFontColor=headerfontcol,
	% Only simple background color used, no shading, so boxColorTwo isn't necessary
	boxColorOne=boxcolor,
	headershape=roundedright,
	headerfont=\large\sf\bf,
	textborder=faded,%rectangle,
	headerborder=open,
  	boxshade=plain,
	background=plain,%shadetb,
	bgColorOne=white,
	bgColorTwo=blue!10,
	headerheight=6.5cm
}
%%% Eye Cacther %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
{

	\includegraphics[width=3cm]{figures/Max-Planck-Gesellschaft.png}

}
%%% Title %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
{\Large\bf
	MitoBenchStarter: An interactive visual workbench for population genetics on mitochondrial DNA
}
%%% Authors %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
{
	\vspace{1em} Judith Neukamm$^{1,3*}$, Alexander Peltzer$^{1,2,3*}$, Wolfgang Haak$^{2}$, Johannes Krause$^{1,2}$ and Kay Nieselt$^{3}$\\
	{\footnotesize 1 Institute for Archaeological Sciences, Archaeo- and Paleogenetics, University of Tuebingen, Germany.\\
	2 Max Planck Institute for the Science of Human History, Jena, Germany.\\
	3 Integrative Transcriptomics, Center for Bioinformatics (ZBIT), University of Tuebingen, Germany.\\
	$*$These authors contributed equally to the study.\\
	\vspace{1em}
	Contact: judith.neukamm@uni-tuebingen.de
	}
}
%%% Logo %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
{
% The logos are compressed a bit into a simple box to make them smaller on the result
% (Wasn't able to find any bigger of them.)
%\setlength\fboxsep{0pt}
%\setlength\fboxrule{0.5pt}

\begin{minipage}{0.15\textwidth}
	\begin{center}
		\includegraphics[width=3cm]{figures/Tuebingen.png}\\
		\includegraphics[width=2.5cm]{figures/Pali.png}
	\end{center}
\end{minipage}

	%\fbox{
		%\begin{minipage}{10em}
			%\includegraphics[width=4cm]{figures/it-logo.png}	\\
			%\\
%			\includegraphics[width=3cm]{figures/Max-Planck-Gesellschaft.png}
%

%		\end{minipage}
	%}
}



%----------------------------------------------------------------------------------------
%	Introduction
%----------------------------------------------------------------------------------------

\headerbox{Motivation \& Goals}{name=introduction, row=0, column=0, span=5}{
		\begin{itemize}[leftmargin=*]
			\item Especially in the research field of ancient DNA, mtDNA is often the only proxy available to study extinct populations and their relationship with modern populations
			\item Tools for the analysis typically rely on different file formats $\rightarrow$ require manual interaction with the data for downstream analysis
			\item mitoBench: workbench to interactively analyze and visualize mitochondrial genomes and ease file conversions
		\end{itemize}


%Despite the availability of modern next generation sequencing technologies and
%therefore nuclear human genomes, the sequencing and analysis of mitochondrial DNA
%(mtDNA) is still common. Especially in the research field of ancient DNA and the
%context of population genetics, mtDNA is often the only proxy available to study
%extinct populations and their relationship with modern populations. As a consequence,
%many population genetic studies rely on the analysis of mtDNA. \\
%A plethora of methods for the analysis of mtDNA exist, that address questions in
%population genetics, phylogeny and others. However, these tools typically rely on
%different file formats and often require manual interaction with the data for downstream
%analysis. Ultimately, these steps can be cumbersome, especially for non-bioinformaticians,
%resulting in an increased risk of errors during the analysis. \\
%To tackle these issues, we present MitoBenchStarter, a workbench to interactively analyze
%and visualize mitochondrial genomes with a focus on population genetics. The
%graphical user interface is kept simple, to accommodate even users without further
%prior knowledge on computational methods. Furthermore, it shows additional
%information such as metadata and statistics. Currently, MitoBenchStarter offers automatic
%file conversion tools to connect the workbench with existing analysis methods
%such as BEAST, Arlequin and others.

}






%----------------------------------------------------------------------------------------
%  Workflow
%----------------------------------------------------------------------------------------


\headerbox{Workflow}{name=workflow, column=0, span=2, below=introduction}{
	\centering
	\includegraphics[width=0.7\textwidth]{figures/workflow2.jpg}

}

%----------------------------------------------------------------------------------------
%	MitoDB
%----------------------------------------------------------------------------------------


\headerbox{MitoDB}{name=mitodb, row=1, column=2, span=3, below=introduction, bottomaligned=workflow}{

	\begin{minipage}{0.8\textwidth}
		\begin{itemize}
			\item Web-Frontend with Vaadin Java Framework
			\item Backend with PostgreSQL, providing sequence information and meta-data (SQL)
			\item Curated data upload with metadata
			\item Retrieval possible through WebUI and mitoBench
		\end{itemize}
	\end{minipage}
	\begin{minipage}{0.15\textwidth}
		\begin{center}
			\includegraphics[width=2cm]{figures/postgresql.png}
			\vspace{2em}
			\includegraphics[width=0.1\paperwidth]{figures/vaadin.png}
		\end{center}
	\end{minipage}

}



%----------------------------------------------------------------------------------------
%	Mitobench
%----------------------------------------------------------------------------------------


\headerbox{MitoBenchStarter}{name=mitobench, column=0, span=3,below=workflow}{
	\begin{minipage}[t]{0.5\textwidth}
		\textbf{Data import and export}\\
		\includegraphics[width=0.8\textwidth, left]{figures/import_export.jpg}
	\end{minipage}
	\hspace{0.5em}
	\begin{minipage}[t]{0.5\textwidth}
		\textbf{Data representation}\\
		\\
		\includegraphics[width=0.85\textwidth, left]{figures/load_generic.png}
	\end{minipage}

	\vspace{2em}

	\begin{minipage}[t]{0.5\textwidth}
		\textbf{Data grouping}
		\begin{itemize}[leftmargin=*]
			\item Interested in behaviour of different groups\\
			$\rightarrow$ Group data by feature
			\item Internal grouping (columns not sorted)
		\end{itemize}
	\end{minipage}
	\hspace{0.5em}
	\begin{minipage}[t]{0.5\textwidth}
		\textbf{Data filtering / Statistics}
		\begin{itemize}[leftmargin=*]
			\item Haplogroup filtering / frequencies
			\item Mutation filtering / frequencies
		\end{itemize}
		Haplogroups based on \includegraphics[width=2cm]{figures/phylotree.png}
	\end{minipage}

	\vspace{2em}

	\begin{minipage}[t]{0.5\textwidth}
		\textbf{Data visualizations}\\
		\\
		\includegraphics[width=\textwidth, left]{figures/stackedBarchart.png}
	\end{minipage}
	\hspace{0.5em}
	\begin{minipage}[t]{0.5\textwidth}
		\textbf{}\\
		\\
		\includegraphics[width=0.9\textwidth, left]{figures/profile.png}
	\end{minipage}

	\vspace{2em}

	\begin{minipage}[t]{0.5\textwidth}
		\textbf{}\\
		\includegraphics[width=0.85\textwidth, center]{figures/sunburst2.png}
	\end{minipage}
	\hspace{0.5em}
	\begin{minipage}[t]{0.5\textwidth}
		\textbf{}\\
		\\
		\includegraphics[width=0.9\textwidth, left]{figures/group_sizes.png}
	\end{minipage}

}



%----------------------------------------------------------------------------------------
%	Conclusions
%----------------------------------------------------------------------------------------

\headerbox{Conclusion}{name=conclusions,row=0, column=3, span=2,  below=mitodb}{
	\textbf{mitoBench}
	\begin{itemize}
		\item Easy file handling and file conversion
		\item Combination, manipulation and \\ visualization of mitochondrial data
		\item Data filtering
		\item Statistics
	\end{itemize}
	\textbf{mitoDB}
	\begin{itemize}
		\item Collection of mitochondrial data \\ plus meta information
		\item Curated database
		\item Accessible via mitoBench and web interface
	\end{itemize}

}

%----------------------------------------------------------------------------------------
%	Outlook
%----------------------------------------------------------------------------------------

\headerbox{Outlook}{name=outlook,row=0, column=3, span=2,  below=conclusions, bottomaligned=mitobench}{
	\textbf{mitoBench}
	\begin{itemize}
		\item Providing methods for downstream analysis (e.g. F$_{st}$ calculations, Founder Analysis)
		\item More visualizations (e.g. using maps for geographical information)
	\end{itemize}
	\textbf{mitoDB}
	\begin{itemize}
		\item Workspace: Potentially some analysis inside the WebUI
		\item Export/import functionality
	\end{itemize}

\vspace{3em}
\begin{flushright}
	\begin{minipage}{0.75\textwidth}
			\textbf{Looking for more information?}\\
			%\phantom{x} TAP \hspace{1em} or \hspace{4em} SCAN\\
			\includegraphics[width=2cm]{figures/nfc_logo.png} %\hspace{1em}
			\includegraphics[width=2cm]{figures/qr_code_references.jpg} %\hspace{1em}
			\includegraphics[width=2cm]{figures/qr_code_references.jpg} \\
			\phantom{x} \hspace{2em} Poster guide \hspace{3em} References
	\end{minipage}
\end{flushright}

}






%----------------------------------------------------------------------------------------
%	References
%----------------------------------------------------------------------------------------

%\headerbox{References}{name=references, column=0, span=5, above=bottom, below=mitobench}{
%	\footnotesize
%	\begin{enumerate}[topsep=0pt,itemsep=-1ex,partopsep=1ex,parsep=1ex]
%		%\item Drummond, Alexei J. et al. "Bayesian phylogenetics with BEAUti and the BEAST 1.7." Molecular biology and evolution 29.8 (2012): 1969-1973.
%		%\item Excoffier, Laurent,  and EL Lischer, Heidi. Arlequin suite ver 3.5: a new series of programs to perform population
%		%genetics analyses under linux and windows. \textit{Molecular ecology resources}, 10(3):564-567, 2010.
%		%\item Weissensteiner, Hansi, et al."Haplogrep 2: mitochondrial haplogroup
%		%classification in the era of high-throughput sequencing." \textit{Nucleic acids research}, page gkw233, 2016.
%		\item Van Oven, Mannis, and Manfred Kayser. "Updated comprehensive phylogenetic tree of global human mitochondrial DNA variation." Human mutation 30.2 (2009): E386-E394.
%	\end{enumerate}
%}


\end{poster}
\end{document}
